Epoch 1
-------------------------------
Train loss = 0.140987  [    8/  540]
Train loss = 0.111919  [   88/  540]
Train loss = 0.096130  [  168/  540]
Train loss = 0.105308  [  248/  540]
Train loss = 0.098775  [  328/  540]
Train loss = 0.239592  [  408/  540]
Train loss = 0.101185  [  488/  540]
Traceback (most recent call last):
  File "C:\Users\tzhan\PythonWS\10423_Spring2024\hw0\handout\img_classifiery.py", line 185, in <module>
    main(args.n_epochs, args.batch_size, args.learning_rate)
  File "C:\Users\tzhan\PythonWS\10423_Spring2024\hw0\handout\img_classifiery.py", line 163, in main
    evaluate(train_dataloader, "Train", model, loss_fn, t, n_epochs)
  File "C:\Users\tzhan\PythonWS\10423_Spring2024\hw0\handout\img_classifiery.py", line 125, in evaluate
    for i, X, y in enumerate(dataloader):
        ^^^^^^^
ValueError: not enough values to unpack (expected 3, got 2)